dataservice/util/data_gen/data_generator.py

```diff
@@ -0,0 +1,316 @@
+            self.p = Participant(
+                external_id='participant_{}'.format(i),
+                samples=self._create_samples(
+                    random.randint(
+                        self.min_samples,
+                        self.max_samples)),
+                demographic=self._create_demographics(i),
+                diagnoses=self._create_diagnoses(
+                    random.randint(
+                        self.min_samples,
+                        self.max_diagnoses)))
+            db.session.add(self.p)
+        db.session.commit()
+
+    def _create_samples(self, total):
+        """
+        Create samples with aliquots
+        """
+
+        self.s_list = []
+        for i in range(total):
+            self.sample_data = {
+                'external_id': 'sample_{}'.format(i),
+                'tissue_type': random.choice(self.tissue_type_list),
+                'composition': random.choice(self.composition_list),
+                'anatomical_site': random.choice(self.anatomical_site),
+                'age_at_event_days': random.randint(0, 32872),
+                'tumor_descriptor': random.choice(self.tumor_descriptor_list)
+            }
+            self.s_list.append(
+                Sample(
+                    **self.sample_data,
+                    aliquots=self._create_aliquots(
+                        random.randint(
+                            self.min_aliquots,
+                            self.max_aliquots))))
+        return self.s_list
+
+    def _create_aliquots(self, total):
+        """
+        Creates aliquots with sequencing experiments
+        """
+        self.dt = datetime.now()
+        self.a_list = []
+        for i in range(total):
+            self.aliquot_data = {
+                'external_id': 'aliquot_{}'.format(i),
+                'shipment_origin': random.choice(self.shipment_origin_list),
+                'shipment_destination':
+                    random.choice(self.shipment_destination_list),
+                'analyte_type': random.choice(self.analyte_type_list),
+                'concentration': random.randint(70, 400),
+                'volume': (random.randint(200, 400)) / 10,
+                'shipment_date': self.dt - relativedelta.relativedelta(
+                    years=random.randint(1, 2)) - relativedelta.relativedelta(
+                    months=random.randint(1, 12)) +
+                relativedelta.relativedelta(days=random.randint(1, 30))
+            }
+            self.a_list.append(
+                Aliquot(
+                    **self.aliquot_data,
+                    sequencing_experiments=self._create_experiments(
+                        random.randint(self.min_seq_exps, self.max_seq_exps))))
+        return self.a_list
+
+    def _create_experiments(self, total):
+        """
+        Creates sequencing experiments
+        """
+
+        self.e_list = []
+        dt = datetime.now()
+        for i in range(total):
+
+            self.e_data = {
```

Comment:
Could do with just assigning to `e_data` rather than a class property. I'm guessing you never actually need `self.e_data`?